https://gitlab.archlinux.org/archlinux/mkinitcpio/mkinitcpio/-/merge_requests/259

From 889b1c39653e66b5492b01cd34892506a9a5e9ee Mon Sep 17 00:00:00 2001
From: classabbyamp <dev@placeviolette.net>
Date: Tue, 12 Sep 2023 22:31:36 -0400
Subject: [PATCH] install/consolefont: include setfont binary

if not using the sd-vconsole hook (like on Void Linux, where there is no
systemd), setfont is not available unless manually added to BINARIES.
This also ensures this hook is fully modular and does not rely on other
hooks to work.
---
 install/consolefont | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/install/consolefont b/install/consolefont
index 0ec554e1..36451912 100644
--- a/install/consolefont
+++ b/install/consolefont
@@ -6,6 +6,8 @@ build() {
     # shellcheck disable=SC2034
     local KEYMAP KEYMAP_TOGGLE FONT FONT_MAP FONT_UNIMAP XKBLAYOUT XKBMODEL XKBVARIANT XKBOPTIONS
 
+    add_binary setfont
+
     # subshell to avoid namespace pollution
     (
         # shellcheck disable=SC1091
-- 
GitLab

